### ISSUE TITLE
Why is there difference in early 24 epochs of scheduler 3x and scheduler 4x

### ISSUE THREAD

**Issue by childi** [open-mmlab/mmdetection3d#1814]
I trained a network with scheduler 3x and 4x, respectively. All other hyper-parameters are the same, including learning rate, momentum and etc.

But why is there obvious performance gap in early 24 epochs. Instinctively, since they share the same learning rate, the results should be the same too. But it seems scheduler 4x is slowly rising and scheduler 3x is fluctuating.

As shown below,  orange represents 3x and grey represents 4x. And epoch 24 == 1.43k steps.

![1662552385520](https://user-images.githubusercontent.com/17497971/188874472-1187e9b8-6f2b-41a5-92cb-e9f926b9a4e2.jpg)

**Comment by Tai-Wang** [comment]
The detection performance of pedestrians is unstable on KITTI. From my perspective, this fluctuation is actually normal. Did you observe this phenomenon also for cars and in multiple runs?
